lib/taste_tester/config.rb

```diff
@@ -52,6 +52,7 @@ module Config
     use_ssl true
     chef_zero_logging true
     chef_config_path '/etc/chef'
+    no_ipv6 false
```

Comment:
double negatives kill puppies.

make the option `ipv6` and have it default to true.